package.json

```diff
@@ -51,15 +51,15 @@
   ],
   "dependencies": {
     "es6-promise": "~3.0.2",
-    "restler": "~3.3.0",
+    "restler": "git+https://github.com/piperamirez/restler.git",
```

Comment:
Why does it need to go to your own fork?

Reply:
Sorry my bad. I opened the PR from master in my fork where there are other fixes. I'll open another one with just the feature. Closing now.